scipy/special/_add_newdocs.py

```diff
@@ -763,11 +763,11 @@ def add_newdoc(name, doc):
     >>> (binom(x, y), comb(x, y), comb(x, y, exact=True))
     (960566918219.9999, 960566918219.9999, 960566918220)
 
-    Note that if ``exact=True``, non-integer arguments to  `comb` will be
```

Comment:
I think it's fine to just get rid of this example. I'd written it to draw attention to the strange behavior for non-integral arguments when `exact=True` which is being fixed now :) If we just get rid of it, then we don't have to worry about remembering to remove it when the `legacy` keyword is removed.